natural-compare.js

```diff
@@ -77,6 +77,14 @@ function naturalCompare(a, b) {
     ++bIndex;
   }
 
+  if (aIndex >= lengthA && bIndex < lengthB && a.length >= b.length) {
+    return -1;
+  }
+
+  if (bIndex >= lengthB && aIndex < lengthA && b.length >= a.length) {
```

Comment:
Instead of `a.length` and `b.length`, please use `lengthA` and `lengthB`.

Reply:
ok